source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -166,8 +166,15 @@ namespace PlayFab
         {
             try
             {
+                size_t sizeOfBatchesInFlight = 0;
+
+                { // LOCK batchesInFlight mutex
+                    std::unique_lock<std::mutex> lock(inFlightMutex);
+                    sizeOfBatchesInFlight = batchesInFlight.size();
```

Comment:
>batchesInFlight [](start = 44, length = 15)

probably needs decorated with the unnecessary `this->` to be consistent with the rest of the code.